bin/homey.js

```diff
@@ -15,16 +15,17 @@ try {
   version = version.split('.');
 
   let majorVersion = version[0];
-  majorVersion = parseInt(majorVersion);
+  majorVersion = parseInt(majorVersion, 10);
 
   let minorVersion = version[1];
-  minorVersion = parseInt(minorVersion);
+  minorVersion = parseInt(minorVersion, 10);
 
   if (majorVersion < 10 || (majorVersion === 10 && minorVersion < 10)) {
-    return console.error(`Error: node-homey requires Node.js v${REQUIRED_NODE_VERSION} or higher to run. Please upgrade your Node.js version and try again.`);
+    console.error(`Error: node-homey requires Node.js v${REQUIRED_NODE_VERSION} or higher to run. Please upgrade your Node.js version and try again.`);
```

Comment:
The PR desc. states the minimum node version is increased, this should also be reflected in `REQUIRED_NODE_VERSION` in this file, and the if statement directly above this line.

Reply:
see https://github.com/athombv/node-homey/pull/203